BrainPortal/app/views/shared/_access_profile_checkbox_table.html.erb

```diff
@@ -0,0 +1,50 @@
+
+<%-
+#
+# CBRAIN Project
+#
+# Copyright (C) 2008-2012
+# The Royal Institution for the Advancement of Learning
+# McGill University
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+-%>
+
+<%
+   checked    ||= []
```

Comment:
Andrew, obviously you used the `_users_checkbox_table.html.erb` file as a template to generate this one, but why did you remove all the comments that explained what this template was about and what local variables it expects? You should have kept these comments and adjusted them appropriately... Right now, looking at this file, we have no idea what the variable named `variable_name` (line 47) is or where it comes from.